tools/rotate-certs.sh

```diff
@@ -69,3 +69,16 @@ ${_kubectl} delete pods --namespace "${namespace}" -l kubevirt.io
 echo "# Rotating SSP certificates ..."
 ${_kubectl} delete secrets --ignore-not-found=true --namespace "${namespace}" virt-template-validator-certs
 ${_kubectl} delete pods --namespace "${namespace}" -l kubevirt.io=virt-template-validator
+
+echo "# Rotating Web UI certificates ..."
+oc get crd kwebuis.kubevirt.io >/dev/null
+if [ $? -eq 0 ]; then
+  # valid for 1.4.Z only
+  formerVersion=$(${_kubectl} get kwebui kubevirt-web-ui -o yaml | grep '  version: '|sed 's/^.*: *\(.*\)$/\1/g')
+  echo Detected former Web UI version: ${formerVersion}
+  ${_kubectl} patch kwebui kubevirt-web-ui --patch '{"spec": {"version": ""}}' --type=merge # undeploy
+  sleep 15 # give kubevirt-web-ui-operator enough time to react, but it's usually instant
```

Comment:
An unconditional sleep is not really good. You are probably waiting for the ui-deployment to be scaled down or deleted. Could you either just scale down the deployment (which is waiting), or periodically check that the deployment got deleted?

Reply:
Actually I already had code here checking presence of pods (they are about to be terminated). But found that this is not important. The operator just needs to get time to initiate the reconciliation - to start undeployment. We do not need to wait for this undeployment to be finished.

If really requested, I can add similar check back, either based on status in the CR or pod presence, but it will make the code here harder to read with no/very limited benefit.

Reply:
What if the undeploy does not happen withing 15 seconds (e.g. temporary error or rbac issue, or whatever)?  You would set the version again after 15 seconds. Wouldn't you then risk that no rotation happened at all?

Reply:
Ok, wating is added